jck/compiler.api/playlist.xml

```diff
@@ -24,12 +24,6 @@
 		$(TEST_STATUS); \
 		$(GEN_SUMMARY_GENERIC) testsuite=COMPILER tests="$(JCKCOMPILER_CUSTOM_TARGET)" isCustomTarget="isCustomTarget"
 		</command>
-		<disables>
-			<disable>
-				<comment>Temp exclude for Jul_2025 release, if testcases fail, auto-reruns will fail</comment>
-				<impl>hotspot</impl>
-			</disable>
-		</disables>
```

Comment:
this was temporarily excluded for July, and TC team asked us to re-enable, they can communicate to us during dry run if any concerns regarding this change